shadows/framework/src/main/java/org/robolectric/shadows/ShadowSettings.java

```diff
@@ -350,7 +348,7 @@ public static class ShadowGlobal {
     private static final ImmutableMap<String, Optional<Object>> DEFAULTS =
         ImmutableMap.<String, Optional<Object>>builder()
             .put(Settings.Global.ANIMATOR_DURATION_SCALE, Optional.of(1))
-            .buildOrThrow();
+            .build();
```

Comment:
Doesn't `ImmutableMap.of()` work here?
```java
ImmutableMap.of(Settings.Global.ANIMATOR_DURATION_SCALE, Optional.of(1));
```

Reply:
Ah yeah that wasn't caught. I can change that. Sorry for the confusing CL title, I will update it in the Google branch